src/routes/backpack/base.php

```diff
@@ -13,7 +13,7 @@
 Route::group(
 [
     'namespace'  => 'Backpack\CRUD\app\Http\Controllers',
-    'middleware' => 'web',
```

Comment:
@MartijnvAdrichem I'm wondering. If you need this changed in your Base because you don't want the "web" middleware.

Don't you need it in your CRUD routes too? Shouldn't we use ```config('backpack.base.route_middleware')``` instead of ```web``` here too? https://github.com/Laravel-Backpack/CRUD/blob/master/src/routes/backpack/custom.php#L11